dialogue/local-abundance.md

```diff
@@ -0,0 +1,45 @@
+S: Surprise, surprise… Guess who’s here?
+
+P: Surprise?
+
+S: I’m not sure about that. I have a question for you, mind if I ask?
+
+P: Sure, go ahead.
+
+S: How is it possible that your name is Play, yet I sense some sadness and fatigue in your tone?
+
+P: That’s a great question – it’s quite counterintuitive, actually. Just because my name is Play, people assume my life is like a blossoming garden. I don’t see it that way. I look at myself more like water.
+
+S: Water?
+
+P: Yes, water. There’s something about water that makes it so special and unique. It’s hard to describe it with words – it’s more of a feeling, you know? Our [terminology](https://jzhao.xyz/thoughts/terminology/) isn’t quite there yet. Nevertheless, I’ll explain to you why water is something so close to my heart. Growing up, I used to take myself very seriously. It was definitely not fun and lasted for a couple of years. My mom eventually took me to this psychologist who diagnosed me with something called “atelophobia”. What do psychologists know about life anyway? I’ve had to figure out on my own how to get out of that bubble.
+
+S: What did you do?
+
+P: I became like water.
+
+S: How did you become like water?
+
+P: You are a very curious person, I’m starting to like you. Growing up with the Internet, I started to understand I could reverse engineer my life and become whatever I wanted to. Once I realized that there’s not really any limitation, I decided to become like water.
+
+S: That’s a peculiar choice.
+
+P: I’ve always looked at life as a choice between intellectual honesty and ego. Intellectual honesty is just a fancy way to say “play”. Ego is everything we do to seek status. Just by observing adults, I realized many of them are just seeking status. Honestly, they look miserable; I’m quite good at spotting sad people. Kids, on the other hand, have this playful view of the world. It’s something very intrinsic and doesn't come from ego. Ego is the worst thing in life, it destroys everything. People usually develop an ego when they start compromising on things they love in order to do things they don’t want to do. Eventually, this “ego” thing starts growing, and people start pretending to love what they do. That’s why I became water. Water has no ego and seeks no [status](https://twitter.com/NavalismHQ/status/1557575533730430976). Water flows.
```

Comment:
yes! Intellectual 'play' turns out to have epistemic functionality as well https://jzhao.xyz/thoughts/Jestermaxxing

Reply:
wow, this is amazing -- I haven't read it before! I've added your link. Intellectual playfulness is a really great concept, it really fits in with what I was trying to say!